docs/components/neural_ranking.rst

```diff
@@ -41,6 +41,30 @@ Further the trained representation-based model can be run for inference over the
 
 Statistics on the models quality are available :doc:`here </intro/features>`.
 
+Building the own response base for bert ranking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the BERT-based models we have the following mechanism of building the own response base.
+If you type `python -m deeppavlov interact ranking_ubuntu_v2_bert_sep_interact.json -d` in console
+the model with the existing base will be downloaded and loaded.
+If you look in the folder where the model with the base is located you will find four files:
+`contexts.csv`, `responses.csv`, `cont_vecs.npy`, `resp_vecs.npy`.
```

Comment:
in `.rst` files you should use \`\`text\`\` instead of \`text\` to get mono-spaced font

Reply:
done